src/wrapped_radix_engine_toolkit/low_level/transaction_manifest/blobs/blobs.rs

```diff
@@ -6,6 +6,29 @@ pub struct Blobs {
     pub(crate) secret_magic: BlobsSecretMagic,
 }
 
+impl Serialize for Blobs {
+    #[cfg(not(tarpaulin_include))] // false negative
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        self.secret_magic.secret_magic.serialize(serializer)
+    }
+}
+
+impl<'de> Deserialize<'de> for Blobs {
+    #[cfg(not(tarpaulin_include))] // false negative
+    fn deserialize<D: Deserializer<'de>>(
+        deserializer: D,
+    ) -> Result<Self, D::Error> {
+        let option_blobs: Option<Vec<Blob>> =
+            Option::deserialize(deserializer)?;
+        let blobs = option_blobs.unwrap_or_default();
```

Comment:
Default is empty, right? Maybe worth having a separate function like `unwrap_or_empty`, to avoid the case when `default` might have a non empty value for some reason.